packages/hash/frontend/src/shared/layout.tsx

```diff
@@ -0,0 +1,26 @@
+import { NextPage } from "next";
+import { ReactElement, ReactNode } from "react";
+import { LayoutWithHeader } from "./layout/layout-with-header";
+import { LayoutWithSidebar } from "./layout/layout-with-sidebar";
+import { SidebarContextProvider } from "./layout/layout-with-sidebar/sidebar-context";
```

Comment:
Minor notice: we are importing a ‘private’ resource here. Not to worry about at this point.

Reply:
Thanks for catching! Fixed here => [125801f](https://github.com/hashintel/hash/pull/468/commits/125801f3662420b4370853247fe978a71cf6d7f2)